pyomo/core/tests/unit/test_action.py

```diff
@@ -63,6 +63,8 @@ def test_getattr(self):
         self.assertEqual( self.instance.A.value, 4.3)
 
 
+Simple = Scalar
```

Comment:
This is a test suite, I don't think we need the "backwards compatibility" alias.